Makefile

```diff
@@ -5,12 +5,12 @@ project := xlang
 install:
 	glide --version || go get github.com/Masterminds/glide
 	GO15VENDOREXPERIMENT=1 glide install
-	GO15VENDOREXPERIMENT=1 go build `GO15VENDOREXPERIMENT=1 glide novendor`
+	GO15VENDOREXPERIMENT=1 go build `glide novendor`
```

Comment:
GNU Make allows for top-level env var's? I didn't realize that... or did you just mean something like:

```
env:
    export GO15VENDOREXPERIMENT=1

install: env
    ...
```